basic/exercise7/script7.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+function calculateDaysUntilToNextChristmas() {
```

Comment:
It does not specify what it returns exactly. What calculated days means?
It should be starting with getNumberOfDays...